models/dydx/ethereum/dydx_ethereum_flashloans.sql

```diff
@@ -14,23 +16,23 @@
 WITH flashloans AS (
     SELECT d.evt_block_time AS block_time
         , d.evt_block_number AS block_number
-        , CAST(get_json_object(get_json_object(d.update, '$.deltaWei'), '$.value') AS DECIMAL(38,0)) AS amount_raw
+        , CAST(json_extract_scalar(JSON_PARSE(json_extract_scalar(d."update", '$.deltaWei')), '$.value') as double) AS amount_raw
```

Comment:
on dunesql, should we cast this as `uint256` instead? similar to `dex.trades` and `nft.trades` lineage?

this would require the casting change on all models in the flashloans lineage

cc @hildobby

Reply:
may be worthwhile to quickly scan through each column and double check types. i think leaving `fee` as double makes sense for instance.

Reply:
ah, i see, since you are dividing later in the query, it becomes double anyway

Reply:
no problem -- we should be good then